src/main/java/Command.java

```diff
@@ -0,0 +1,14 @@
+public class Command {
+    Tasks[] tasks;
+    Command(Tasks[] tasks){
+        this.tasks = tasks;
+    }
+
+    public static void displayLine(){
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addedMessage(){
```

Comment:
The name of a method better is an action. So maybe this method should be called "displayAddedMessage"?